vernac/metasyntax.ml

```diff
@@ -1669,8 +1659,8 @@ let add_notation_interpretation_core ~local df env ?(impls=empty_internalization
     ninterp_rec_vars = to_map recvars;
   } in
   let (acvars, ac, reversibility) = interp_notation_constr env ~impls nenv c in
-  let plevel = match level with Some (from,level,l) -> level | None (* numeral: irrelevant )*) -> 0 in
-  let interp = make_interpretation_vars recvars plevel acvars (List.combine mainvars i_typs) in
+  let plevel = match level with Some (from,level,l) -> (from,level,l) | None (* numeral: irrelevant )*) -> (InConstrEntry,0,[]) in
```

Comment:
Can you add a comment explaining why we map to constr in the `None` case? For future reference.

Reply:
It is a dummy value. Maybe "unused thus arbitrary" would be better than "irrelevant"??